src/util.c

```diff
@@ -431,6 +457,189 @@ int write_result_into_log_file(struct ntttcp_test_endpoint *tep)
 	return 0;
 }
 
+int write_result_into_json_file(struct ntttcp_test_endpoint *tep)
+{
+	struct ntttcp_test *test = tep->test;
+	struct ntttcp_test_endpoint_results *tepr = tep->results;
+	char str_temp1[256];
```

Comment:
This var name of "str_temp1" or "str_temp2" is not friendly to read; how about we define specific variable for: computer_name; os_uname_a, and uname_a_escaped.

Reply:
Has been modified.